locale-fork/locale/de/LC_MESSAGES/djangojs.po

```diff
@@ -910,68 +925,207 @@ msgstr ""
+msgid "Your statement was successfully deleted."
+msgstr "Ihre Mitteilung wurde gelöscht."
 
-#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:160
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:13
 msgctxt "kosmo"
-msgid "Classification: "
-msgstr "Klassifikation"
+msgid ""
+"The comment has already been moderated. Your statement could not be saved."
+msgstr "Der Beitrag wurde bereits moderiert. Ihre Mitteilung konnte nicht gespeichert werden. "
 
-#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:161
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:14
 msgctxt "kosmo"
-msgid "AI"
-msgstr "KI"
+msgid "Go to discussion"
+msgstr "Zur Diskussion"
 
-#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:162
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:15
 msgctxt "kosmo"
-msgid " Block"
-msgstr " Blockieren"
+msgid "Comment blocked successfully."
+msgstr "Der Beitrag wurde blockiert."
 
-#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:163
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:16
 msgctxt "kosmo"
-msgid " Unblock"
-msgstr ""
+msgid "Comment unblocked successfully."
+msgstr "Der Beitrag wurde entsperrt."
 
-#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:164
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:17
 msgctxt "kosmo"
-msgid " Add statement"
-msgstr ""
+msgid "Comment highlighted successfully."
+msgstr "Der Beitrag wurde hervorgehoben."
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:18
+msgctxt "kosmo"
+msgid "Comment unhighlighted successfully."
+msgstr "Die Hervorhebung wurde widerrufen."
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:19
+msgctxt "kosmo"
+msgid "Notification archived successfully."
+msgstr "Die Benachrichtigung wurde archiviert."
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:20
+msgctxt "kosmo"
+msgid "Notification unarchived successfully."
+msgstr "Die Archivierung der Benachrichtigung wurde widerrufen."
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:193
+msgid "kosmo"
+msgid_plural "'s {}comment{} has been reported 1 time since it's creation"
+msgstr[0] ""
+msgstr[1] "hat einen {}Beitrag{} verfasst, der seitdem 1 mal gemeldet wurde."
+
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:213
+msgctxt "kosmo"
+msgid "Classification: "
+msgstr "Klassifikation:"
 
-#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:165
+#: apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx:214
 msgctxt "kosmo"
 msgid " Archive"
-msgstr ""
+msgstr "Archiv"
```

Comment:
There is a blank at the beginning and it probably is the verb, so should be "Archivieren". Will check